tmk_core/common/mousekey.c

```diff
@@ -60,17 +61,27 @@ static uint8_t move_unit(void)
 {
     uint16_t unit;
     if (mousekey_accel & (1<<0)) {
-        unit = (MOUSEKEY_MOVE_DELTA * mk_max_speed)/4;
+        unit = (mk_move_delta * mk_max_speed)*2;
     } else if (mousekey_accel & (1<<1)) {
-        unit = (MOUSEKEY_MOVE_DELTA * mk_max_speed)/2;
+        if (mk_move_delta < 10) mk_move_delta++;
+        unit = (mk_move_delta * mk_max_speed);
     } else if (mousekey_accel & (1<<2)) {
-        unit = (MOUSEKEY_MOVE_DELTA * mk_max_speed);
+        if (mk_move_delta > 1) mk_move_delta--;
+        unit = (mk_move_delta * mk_max_speed);
```

Comment:
Adding and removing from the delta works, however it would be a lot better to only allow one increment/decrement per keypress. Right now if you press for a while it will end up near max or min